101 path module (#1845)
kamranahmedse/developer-roadmap: content/roadmaps/107-nodejs/content/105-nodejs-working-with-files/101-path-module.md

> # Path module
> 
> The `path module` provides utilities for working with file and directory paths.
> 
> There is no need to install it. Being part of the Node.js core, it can be used by simply requiring it:
> 
> const path = require('path');
> 
> These are the `path` methods.
> 
> path.basename(), path.dirname(), path.extname(), path.format(), path.isAbsolute(), path.join(), path.normalize(), path.parse(), path.resolve().
> 
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='blue' badgeText='Read' href='https://nodejs.org/api/path.html'>Official Website</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://nodejs.dev/en/learn/the-nodejs-path-module/'>Official Website</BadgeLink>
> <BadgeLink badgeText='Watch' href='https://youtu.be/j95Lwxvi9JY'>Path Module in Node.js</BadgeLink>

Reviewer: ```suggestion
# path module

The `path` module provides utilities for working with file and directory paths. It's built-in to Node.js core and can simply be used by requiring it.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='Official API Reference' href='https://nodejs.org/api/path.html'>Official Website</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Official Docs' href='https://nodejs.dev/en/learn/the-nodejs-path-module/'>Official Website</BadgeLink>
<BadgeLink badgeText='Watch' href='https://youtu.be/j95Lwxvi9JY'>Path Module in Node.js</BadgeLink>
```
